src/client/components/MealList/MealList.jsx

```diff
@@ -11,10 +11,10 @@ const MealList = () => {
   if (!currentMeals.data) {
     return <h1>...Loading</h1>;
   }
-  let displayMeal;
+  let displayMeals;
```

Comment:
"displayMeals" sounds like a function that will do something in order to display the meals, while this is actually just a variable - an array of objects. It is just meals, so keep it down to nouns and differentiate the logic from function naming. 
If display is necessary to emhpasize here for some reason, then it would displayedMeals.

Reply:
🙏🏾 got it,will change.